### PR TITLE
Update Serge.test.js

### PR DESCRIPTION
**Pull request by adkulas**
Took this from https://beautifier.io/

### CODE REVIEW

**Comment by adkulas** [comment]
No idea if eslint doesn't like this. I just did it in the browser

File: test/Serge.test.js [hunk 1]
```diff
@@ -4,18 +4,18 @@ import { ForbiddenCharacters } from '../oslo/Util.js';
 
 describe('Serge parsing', () => {
 
-	const serge = new Serge('/root', 'test', '/root/test', {
```
Comment (pureooze) [inline test/Serge.test.js:7]: I dont think this changed anything other than whitespace.

Comment (adkulas) [reply test/Serge.test.js:7]: I think you're right

Comment (adkulas) [reply test/Serge.test.js:7]: but the spacing does seem off

Comment (adkulas) [reply test/Serge.test.js:7]: some places are 4 space indent some are 2

Comment (pureooze) [reply test/Serge.test.js:7]: What do you mean by "off"? Its  tabs so should be the same size 🤷

Comment (pureooze) [reply test/Serge.test.js:7]: Hmm I dont see that 😕 
image

Comment (adkulas) [reply test/Serge.test.js:7]: yeah I checked out the branch and its all tabs 🤷

**Pull request closed without merging**